jekyll/_cci2/large-language-model-evaluation.adoc

```diff
@@ -0,0 +1,100 @@
+---
+contentTags:
+  platform:
+    - Cloud
+    - Server v4.x
+---
+= Large Language Model evaluation
+:description: Docs page on setting up LLM evaluation in your CI/CD pipeline
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:experimental:
+
+This page describes common methods for running Large Language Model (LLM) evaluations on CircleCI.
```

Comment:
This page describes common methods for testing applications powered by Large Language Models (LLMs) through evaluations.